linux/keyman-config/run-tests.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+PYTHONPATH=.:$PYTHONPATH
+
+if [ -n "$TEAMCITY_VERSION" ]; then
+    if ! pip3 list --format=columns | grep -q teamcity-messages; then
+        pip3 install teamcity-messages
+    fi
+    python3 -m teamcity.unittestpy discover -s tests -p test_*.py
+else
+    python3 -m unittest discover -v -s tests -p test_*.py
+fi
```

Comment:
Thanks for adding unit tests!

Can you update https://github.com/keymanapp/keyman/tree/master/linux#testing

to help other developers get up and running?

Reply:
For developers, is this a matter of adding `unittest` to
https://github.com/keymanapp/keyman/tree/master/linux#linux-requirementssetup
?

Reply:
> For developers, is this a matter of adding `unittest` to
> https://github.com/keymanapp/keyman/tree/master/linux#linux-requirementssetup
> ?
This is part of the default Python package.

Done.